runtime/darwinia/src/lib.rs

```diff
@@ -151,32 +162,26 @@ frame_support::construct_runtime! {
 		Ethereum: pallet_ethereum = 36,
 		EVM: pallet_evm = 37,
 		EthTxForwarder: darwinia_ethtx_forwarder = 38,
-
-		// // Darwinia <> Crab
-		// BridgeKusamaGrandpa: pallet_bridge_grandpa::<Instance1> = 39,
-		// BridgeKusamaParachain: pallet_bridge_parachains::<Instance1> = 40,
-		// BridgeCrabMessages: pallet_bridge_messages::<Instance1> = 41,
-		// BridgeCrabDispatch: pallet_bridge_dispatch::<Instance1> = 42,
-		// CrabFeeMarket: pallet_fee_market::<Instance1> = 43
```

Comment:
Why not keep these comment and give `pallet_message_queue` a new index 44? 

Better safe than sorry, even if the storage of these modules hava been cleared.

Reply:
Only runtime call cares about the index. These won't affect the storage. I think there isn't security issue. Unless there are some on-going scheduled call. But those pallets were removed long time ago.

Reply:
@hackfisher We have some dissicussion internally, what do you think?

Reply:
Both options look good to me. I will probably keep them to ensure we maintain good co-coding conventions, as I will recall why we skipped these indexes in future. Additionally, I encourage others not to skip them by commenting as we have done previously, new contributors may be guided to bad conventions we didn't expect.

However, I do not have reasons to argue against it from a security perspective.